sites/fast-component-explorer/app/explorer.tsx

```diff
@@ -7,17 +7,12 @@ import {
     ViewerCustomAction,
 } from "@microsoft/fast-tooling-react";
 import React from "react";
-import {
-    neutralLayerL1,
-    neutralLayerL2,
-    StandardLuminance,
-} from "@microsoft/fast-components-styles-msft";
+import { StandardLuminance } from "@microsoft/fast-components-styles-msft";
```

Comment:
This should be available from the `fast-components` package - do we need to keep this import?

Reply:
I don't know how that happened. Must have been a bad copy/paste or something. It was supposed to be "@microsoft/fast-components". Thanks for catching that.